pkg/envoy/proxy.go

```diff
@@ -168,6 +170,11 @@ func (p *Proxy) PodMetadataString() string {
 	return fmt.Sprintf("UID=%s, Namespace=%s, Name=%s, ServiceAccount=%s", p.PodMetadata.UID, p.PodMetadata.Namespace, p.PodMetadata.Name, p.PodMetadata.ServiceAccount.Name)
 }
 
+// GetName returns a unique name for this proxy based on the identity and uuid.
+func (p *Proxy) GetName() string {
+	return fmt.Sprintf("%s-%s", p.Identity.String(), p.UUID.String())
```

Comment:
Since `-` is a part of the UUID, I would use a different delimiter for easier parsing.
```suggestion
	return fmt.Sprintf("%s:%s", p.Identity.String(), p.UUID.String())
```

Reply:
good call! fixed!